internal/ethapi/api.go

```diff
@@ -906,6 +906,15 @@ func DoEstimateGas(ctx context.Context, b Backend, args CallArgs, blockNrOrHash
 	}
 	cap = hi
 
+	// Set sender address or use a default if none specified
+	if args.From == nil {
+		if wallets := b.AccountManager().Wallets(); len(wallets) > 0 {
```

Comment:
What happens if we don't have any wallets. Won't we end up repeating the 26 calls below again, each time discovering no address? Shouldn't we default to whatever the missing addr is already here (`0x00`?)?

Reply:
done